framework/global/CMakeLists.txt

```diff
@@ -21,17 +21,24 @@ set(MODULE global)
 
 include(${CMAKE_CURRENT_LIST_DIR}/modularity/modularity.cmake)
 include(${CMAKE_CURRENT_LIST_DIR}/async/async.cmake)
+include(${PROJECT_SOURCE_DIR}/thirdparty/haw_logger/logger/logger.cmake)
```

Comment:
Can you explain to me what does 'haw' mean?

Reply:
It's just a name that doesn't mean anything :)
(hawthorn)